docs/dapp/tools/localnet.mdx

```diff
@@ -0,0 +1,162 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Localnet
+
+For convenient development and testing of your dApps the Oasis team prepared
+both [ghcr.io/oasisprotocol/sapphire-localnet][sapphire-localnet] and
+[ghcr.io/oasisprotocol/emerald-localnet][emerald-localnet] Docker images.
+They will bring you a complete Oasis network stack to your desktop. The
+Localnet Sapphire instance **mimics confidential transactions**, but it does
+not run in a trusted execution environment nor does it require Intel's SGX on
+your computer. The network is isolated from the Mainnet or Testnet and consists
+of a:
+
+- single Oasis validator node with 1-second block time and 30-second epoch,
+- single Oasis client node,
+- single compute node running Oasis Sapphire or Emerald,
+- single key manager node,
+- PostgreSQL instance,
+- Oasis Web3 gateway with transaction indexer and enabled Oasis RPCs,
+- helper script which populates the account(s) for you.
+
+:::note Hardware requirements
+
+You will need at least 16GB of RAM to run the Docker images in addition to your
+machine's OS.
+
+:::
+
+To run the image, execute:
+
+<Tabs>
+<TabItem value="Sapphire">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet
+```
+
+</TabItem>
+<TabItem value="Emerald">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet
+```
+
+</TabItem>
+</Tabs>
+
+After a while, running the `sapphire-localnet` will show you something like:
+
+```
+sapphire-localnet 2024-05-28-git37b7166 (oasis-core: 24.0-gitfb49717, sapphire-paratime: 0.7.3-testnet, oasis-web3-gateway: 5.1.0)
```

Comment:
I don't think devs need duplicated info for Emerald.

Reply:
I think a output with a newer docker version would be good.
In this regard some info to update the docker image, with `docker pull ghcr.io/oasisprotocol/sapphire-localnet:latest` to get the latest version which includes ROFL, could be helpful for devs that used localnet already months ago.